### PR TITLE
Migrate from drone to CircleCI

### PR DESCRIPTION
**Pull request by burnsed**
Passing!

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+machine:
+  node:
+    version: 6.8.1
+  post:
+    # checkout the ci-scripts and install golang
```
Comment (nathanleiby) [inline circle.yml]: nit: can remove this comment

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - npm prune --production
+deployment:
+  all:
+    branch: master
+    owner: Clever
+    commands:
+      # Publish to dockerhub 
+      - ./ci-scripts/docker-publish $DOCKER_USER $DOCKER_PASS "$DOCKER_EMAIL" $DOCKER_ORG
+      - ./ci-scripts/report-card $DOCKER_USER $DOCKER_PASS "$DOCKER_EMAIL"
+      - cp launch/* $CIRCLE_ARTIFACTS
```
Comment (nathanleiby) [inline circle.yml]: nit: remove this -- unnecessary

File: .drone.yml [hunk 3]
```diff
@@ -1,30 +0,0 @@
...
-    recipients:
-    - drone@clever.com
-  slack:
-    on_failure: true
-    on_started: false
-    on_success: false
-    webhook_url: $$slack_webhook
-publish:
-  report_card: {}
-  catapult:
```
Comment (nathanleiby) [inline .drone.yml]: Can't merge this until Catapult publish CI script is built out (hopefully by tomorrow)

**Comment by burnsed** [comment]
deployment working too: https://circleci.com/gh/Clever/elasticsearch-toolbox/12

**Comment by nathanleiby** [comment]
🚢

**Pull request merged**